0-ad-history-mod-package/simulation/templates/structures/spart/temple.xml

```diff
@@ -7,6 +7,10 @@
   <Identity>
     <Civ>spart</Civ>
     <SpecificName>Naos</SpecificName>
+    <History>
+        Sparta, like the rest of Ancient Greece were polytheistic, believing in all the Greek gods. However, thanks to their materialistic society they had special affinity to warlike gods such as Ares and Apollo, however they were eager to please all gods.
+        Because all of Sparta's men were warriors they did not have designated priests, except, however, for their kings, who were considered priests of Zeus.  
```

Comment:
```suggestion
        Because all of Sparta's men were warriors, they did not have designated priests, except, however, for their kings, who were considered priests of Zeus.
```